src/engine/sync/synccontrol.cpp

```diff
@@ -204,8 +201,12 @@ double SyncControl::getBeatDistance() const {
     return adjustSyncBeatDistance(beatDistance);
 }
 
-double SyncControl::getBaseBpm() const {
-    return m_pLocalBpm->get() / m_leaderBpmAdjustFactor;
+mixxx::Bpm SyncControl::getBaseBpm() const {
+    const mixxx::Bpm bpm = getLocalBpm();
+    if (!bpm.isValid()) {
+        return {};
```

Comment:
does {} result in an invalid bpm getting returned?  It would be clearer to explicitly return kInvalidBpm. (that would be correct here)

Reply:
Yes, this returns a default constructed Bpm (it's the same as `mixxx::Bpm()`, but repeating the return type is not that nice). 

There is no `kInvalidBpm`. We could add it if you think itvs necessary.